osgp/platform/osgp-adapter-domain-smartmetering/pom.xml

```diff
@@ -159,6 +159,13 @@
       <artifactId>logback-core</artifactId>
     </dependency>
 
+    <!-- Lombok -->
+    <dependency>
+      <groupId>org.projectlombok</groupId>
+      <artifactId>lombok</artifactId>
+      <scope>provided</scope>
```

Comment:
is it provided? by what ?

Reply:
Only needed compile-time. Scope changed to compile